docs/source/03_getting_started/01_quickstart.md

```diff
@@ -54,6 +54,17 @@ Change directory to the project generated in /home/mario/kedro/kubeflow-plugin-d
 A best-practice setup includes initialising git and creating a virtual environment before running `kedro install` to install project-specific dependencies. Refer to the Kedro documentation: https://kedro.readthedocs.io/
 ```
 
+There are some adjustments that need to be made to run starter Kedro project on Kubeflow. We need to replace all dots in names of catalog with other characters as Kubeflow does not accept dots in names. You can do it by using `sed` in starter root directory:
```

Comment:
This is not true, currently end 2 end tests are passing with catalog entries having dots. What I would mention instead is that the plugin was not yet adjusted to work with [kedro namespaces](https://kedro.readthedocs.io/en/stable/tutorial/namespace_pipelines.html) so using them may cause issues.